packages/clay-css/src/scss/mixins/_tables.scss

```diff
@@ -366,6 +366,37 @@
 				}
 			}
 
+			.autofit-col-checkbox {
+				@include clay-css(map-get($map, autofit-col-checkbox));
+			}
+
+			.autofit-col-icon {
+				@include clay-css(map-get($map, autofit-col-icon));
+			}
```

Comment:
I don't really like that we add classes like this to deal with spacing between elements, I wish we used more of the spacing classes we have instead this improves reusability and in the future we can have more use of components like `<Stack/ >` which would handle whitespace between elements following the standards.

Reply:
@matuzalemsteles I was debating with myself whether it should be utilities like 1-10 or this above. I chose easier for the theme developer over component developer. My reasoning is checkbox and icon are the same width (currently 16px), but in the future if we want one of them to be larger (24px), it's really hard to target the column with utilities. It will have to be something like 
```css
.table-nested-rows .autofit-col-1 + .autofit-col-1 {}
````

Reply:
Yes that makes sense too. But we have to understand that the utilities follow a spacing pattern that would be defined by the Lexicon, its white space pattern. For example, we will not see a pattern outside 1.5px but rather 4px, 2px... that is, if a theme developer wants to change the spacing of his application to apply different styles or have more white space instead of doing this to Each component has its own space palette which in theory will reflect this for all components, what we have to think about is that this has to be consistent so that the UI does not create inconsistency of different spaces for each component and this applies to any UI is a theme developer as well as improving the UI for the end user and less maintenance. Does this make sense to you?

Reply:
@matuzalemsteles Ok how about we use spacing utils with specific column names? We don't need to use them, but can tie them in with spacing utils if the need arises.

Reply:
Hmm what would that be like? I'm not sure I understand correctly.

Reply:
It would be something like `class="autofit-col-1 checkbox-column"`.

Reply:
I see. I'm not sure, we can go with that for now then and think more about it. What do you think? so we can discuss it more.

Reply:
Ok thanks, I'll send an update today.